packages/zoe/src/zoeService/types.js

```diff
@@ -112,12 +112,19 @@
  * @typedef {any} OfferResult
  */
 
+/**
+ * @typedef {Object} AdminFacet
+ * @property {() => Promise<string|Error|any>} getVatShutdownPromise
```

Comment:
```suggestion
 * @property {() => Promise<Completion>} getVatShutdownPromise
```

The type parameter of `Promise` is only for the fulfillment type. The rejection reason is normally an error, but is in any case not stated in the promise type.

Reply:
done